SchibstedAccount/Manager/User/User.swift

```diff
@@ -124,6 +134,40 @@ public class User: UserProtocol {
         log(from: self, "removed User \(ObjectIdentifier(self).hashValue) from global store")
     }
 
+    /**
+     Checks whether the user has accepted the latest terms and conditions.
+
+     Since new terms and conditions may be issued at any time, you should call this method at the app's startup to check that the logged in user (if any) you
+     have obtained from an instance of `IdentityManager` has accepted the latest terms. If the result provided from the completion callback is `false`, then you
+     should then present a screen where the user can review and accept the updated terms.
+
+     The recommended way of presenting the terms acceptance screen is by using the provided UI flows, thus instantiating `IdentityUI` and then calling
+     `IdentityUI.presentIdentityProcess(from:,route:,identityManager:)` passing `.presentUpdatedTerms` as route. It is *crucial* that you pass the same instance
+     of `IdentityManager` you previouslty used to obtain and store the current logged-in user, otherwise you won't get logout notifications for that user in
+     case the user is logged out for not having accepted the new terms.
+
+     If you are using the headless approach instead, you should then present your own UI and manually call `acceptNewTerms(:)`, if the user accepted the new
+     terms, or `logout()`, if the user rejected them.
+
+     - parameter completion: Callback that is called after acceptance status has been checked.
+     */
+    public func hasAcceptedLatestTerms(completion: @escaping BoolResultCallback) {
+        self.agreements.status(completion: completion)
+    }
```

Comment:
Yeah, maybe just add comments to `AgreementsAPI.status` instead of a wrapper function.

Reply:
Yes, no need for the wrapper 👌